test/org.hdfgroup.object.test/object/DatatypeTest.java

```diff
@@ -16,161 +21,511 @@
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 1, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "String, length = 2, padding = H5T_STR_NULLTERM, cset = H5T_CSET_ASCII",
-            "Variable-length", "Variable-length", "Variable-length", "Variable-length",
-            "Variable-length", "Variable-length", "Variable-length", "Variable-length",
-            "Variable-length", "Variable-length", "Variable-length", "Variable-length",
-            "Variable-length", "Variable-length", "Variable-length"
-    };
+    private String[] descriptions = {"8-bit bitfield",
```

Comment:
This one seems a lot worse, but it's also a lot of stuff to put in one spot

Reply:
Fixed